Base/Python/slicer/util.py

```diff
@@ -2349,25 +2358,40 @@ def errorDisplay(text, windowTitle=None, parent=None, standardButtons=None, **kw
     messageBox(text, parent, windowTitle=windowTitle, icon=qt.QMessageBox.Critical, standardButtons=standardButtons, **kwargs)
 
 def confirmOkCancelDisplay(text, windowTitle=None, parent=None, **kwargs):
-  """Display an confirmation popup. Return if confirmed with OK."""
+  """Display a confirmation popup. Return if confirmed with OK.
+  While testing, this returns True ("Ok") without displaying any popup.
```

Comment:
"While testing" is not clear enough. Instead, you can write "when the application is running in testing mode (`slicer.app.testingEnabled()==True`)...". Update it not just here but in every function that we now update.

Reply:
Last, it may be good to add a new section to the developer guide to briefly describe the testing mode, and use of `slicer.app.testingEnabled()`. See https://github.com/Slicer/Slicer/issues/6164 for more details.